java/test/jmri/jmrix/can/cbus/swing/modeswitcher/SprogCbusModeSwitcherPaneTest.java

```diff
@@ -0,0 +1,63 @@
+package jmri.jmrix.can.cbus.swing.modeswitcher;
+
+import java.awt.GraphicsEnvironment;
+import jmri.GlobalProgrammerManager;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+import jmri.jmrix.can.TrafficControllerScaffold;
+import jmri.jmrix.can.cbus.CbusDccProgrammer;
+import jmri.jmrix.can.cbus.CbusDccProgrammerManager;
+import jmri.jmrix.can.cbus.CbusPreferences;
+import jmri.util.JUnitUtil;
+import org.junit.After;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Tests for the ModeSwitcherPane class
+ *
+ * @author Andrew Crosland (C) 2020
+ */
+public class SprogCbusModeSwitcherPaneTest extends jmri.util.JmriJFrameTestBase {
+
+    CanSystemConnectionMemo memo;
+    CbusDccProgrammer prog;
+    jmri.jmrix.can.TrafficController tc;
+    
+    @Test
+    public void testInitComponents() throws Exception{
```

Comment:
You really don’t need this test.  initComponets is called as part of testShowAndClose in the base class.

Reply:
OK. Again, I think this is copied from another test that I used as a template but I may be confusing between testing JmriJFrames and other panes.